### ISSUE TITLE
Question about RightToLeft languages

### ISSUE THREAD

**Issue by FarisHijazi** [JaidedAI/EasyOCR#882]
Hello, my question is about right to left languages like Arabic, do I have to mirro the data or reverse the text?

I'm finetuning an Arabic model (recognition only) on numbers only.

I'm using the base arabic.zip that is in the releases
I'm using the base configs that come in `configs_all/en_filtered_config.yaml` but I changed the characters of course.

and setting `freeze_FeatureFxtraction: True`

Now my accuracy is much higher before finetuning the model (the base model), and the accuracy gets worse on test set the longer I train

am I doing something wrong?

I'm using generated data from [trdg](https://github.com/Belval/TextRecognitionDataGenerator)
